CITATION.cff

```diff
@@ -0,0 +1,24 @@
+cff-version: 1.2.0
+authors:
+ - name: Preferred Networks, Inc.
+title: "Optuna: A hyperparameter optimization framework"
+url: "https://github.com/optuna/optuna"
+preferred-citation:
+  type: conference-paper
+  authors:
+  - family-names: "Akiba"
+    given-names: "Takuya"
+  - family-names: "Sano"
+    given-names: "Shotaro"
+  - family-names: "Yanase"
+    given-names: "Toshihiko"
+  - family-names: "Ohta"
+    given-names: "Takeru"
+  - family-names: "Koyama"
+    given-names: "Masanori"
```

Comment:
Minor detail: If we apply formatting to YAML, it would look like the following. (Currently, there is no formatter set for YAML files, and it’s passing CI.)

```suggestion
  - name: Preferred Networks, Inc.
title: "Optuna: A hyperparameter optimization framework"
url: "https://github.com/optuna/optuna"
preferred-citation:
  type: conference-paper
  authors:
    - family-names: "Akiba"
      given-names: "Takuya"
    - family-names: "Sano"
      given-names: "Shotaro"
    - family-names: "Yanase"
      given-names: "Toshihiko"
    - family-names: "Ohta"
      given-names: "Takeru"
    - family-names: "Koyama"
      given-names: "Masanori"
```